docs/concepts/pdp/factdb.mdx

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 11
+title: Handling Volume ( EAP )
+---
+
+# FactDB ( EAP )
+
+FactDB is a new feature that allows you to store and query facts more efficiently in the PDP.
+It uses embedded DB technology to store facts in a more efficient way instead of the policy engine memory,
+and provides a RestAPI to query those facts.
+
+:::info EAP
+This feature is in EAP stage - please contact with us if you want to try it out.
```

Comment:
I would edit this note
eap - subject to breaking changes
opt in feature - you should ask to join